lte/cloud/go/services/subscriberdb/servicers/subscriberdb_servicer.go

```diff
@@ -19,59 +19,52 @@ import (
 
 	"magma/lte/cloud/go/services/subscriberdb"
 	"magma/orc8r/cloud/go/orc8r/math"
-
-	"github.com/golang/glog"
-	"github.com/pkg/errors"
-	"github.com/thoas/go-funk"
-	"google.golang.org/grpc/codes"
-	"google.golang.org/grpc/status"
+	"magma/orc8r/cloud/go/syncstore"
 
 	"magma/lte/cloud/go/lte"
 	lte_protos "magma/lte/cloud/go/protos"
 	"magma/lte/cloud/go/serdes"
 	lte_models "magma/lte/cloud/go/services/lte/obsidian/models"
-	"magma/lte/cloud/go/services/subscriberdb/storage"
 	"magma/orc8r/cloud/go/services/configurator"
 	"magma/orc8r/lib/go/protos"
+
+	"github.com/golang/glog"
+	"github.com/golang/protobuf/ptypes"
+	"github.com/golang/protobuf/ptypes/any"
+	"github.com/pkg/errors"
+	"github.com/thoas/go-funk"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 )
 
 type subscriberdbServicer struct {
 	digestsEnabled         bool
 	changesetSizeThreshold int
 	maxProtosLoadSize      uint64
-	resyncIntervalSecs     uint32
-	digestStore            storage.DigestStore
-	perSubDigestStore      *storage.PerSubDigestStore
-	subStore               *storage.SubStore
-	lastResyncTimeStore    *storage.LastResyncTimeStore
+	resyncIntervalSecs     int64
+	store                  syncstore.SyncStoreReader
 }
 
 func NewSubscriberdbServicer(
 	config subscriberdb.Config,
-	digestStore storage.DigestStore,
-	perSubDigestStore *storage.PerSubDigestStore,
-	subStore *storage.SubStore,
-	lastResyncTimeStore *storage.LastResyncTimeStore,
+	store syncstore.SyncStoreReader,
 ) lte_protos.SubscriberDBCloudServer {
 	servicer := &subscriberdbServicer{
 		digestsEnabled:         config.DigestsEnabled,
```

Comment:
Rather than copying the config values over one-by-one, you *can* use type embedding, if you want https://play.golang.org/p/FNzNYbGgQc6